Lib/logging/handlers.py

```diff
@@ -195,10 +202,11 @@ def shouldRollover(self, record):
         """
         if self.stream is None:                 # delay was set...
             self.stream = self._open()
+        pos = self.stream.tell()
```

Comment:
Seems it is not used.

Reply:
You're right, it's related to a fix #116263 and can be removed here.